clients/rge/index.ts

```diff
@@ -0,0 +1,127 @@
+import routes from '#clients/routes';
+import {
+  IRGECompanyCertifications,
+  NomCertificat,
+} from '#models/garant-environement';
+import { Siren, Siret } from '#utils/helpers';
+import { httpGet } from '#utils/network';
+
+/*
+ * Reconnu Garant de l'Environnement (RGE)
+ * https://france-renov.gouv.fr/annuaire-rge
+ */
+
+export enum WORKING_WITH_ENUM {
+  PROFESSIONAL = 'PROFESSIONAL',
+  INDIVIDUAL = 'INDIVIDUAL',
+}
+
+interface IRGEResponse {
+  results: {
+    adresse: string;
+    code_postal: string;
+    code_qualification: string;
+    commune: string;
+    domaine: string;
+    email: string;
+    nom_certificat: NomCertificat;
+    nom_entreprise: string;
+    nom_qualification: string;
+    organisme: string;
+    particulier: Boolean;
+    siret: Siret;
+    site_internet: string;
+    telephone: string;
+    url_qualification: string;
+  }[];
+}
+
+const clientRGE = async (
+  siren: Siren,
+  useCache = true
+): Promise<IRGECompanyCertifications | {}> => {
+  const route = routes.rge.lines;
+  const response = await httpGet(
+    route,
+    { params: { qs: `siret:${siren}*` } },
+    useCache
+  );
+
+  return mapToDomainObject(response.data as IRGEResponse);
+};
+
+export const mapToDomainObject = (
+  rge: IRGEResponse
+): IRGECompanyCertifications => {
+  if (!rge.results.length) {
```

Comment:
We can also throw an http not found here depending on the behaviour we expect on front. Would gladly discuss it by visio

Reply:
i keep it as unresolve till the visio

Reply:
thow new HttpNotFoundError()